src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -597,4 +611,18 @@ public function theOrderShouldHavePaymentState(OrderInterface $order, $orderPaym
             sprintf('Cannot find order with "%s" order payment state in the list.', $orderPaymentState)
         );
     }
+
+    /**
+     * @Then /^there should be(?:| only) (\d+) payment(?:|s)$/
```

Comment:
I feel obligated to prevent situations when someone can't write `There should be only 1 promotion` and has to pluralize the singular 😦